src/Tempest/Framework/Commands/MigrateUpCommand.php

```diff
@@ -28,12 +30,24 @@ public function __construct(
         description: 'Runs all new migrations',
         middleware: [ForceMiddleware::class, CautionMiddleware::class],
     )]
-    public function __invoke(): void
-    {
+    public function __invoke(
+        #[ConsoleArgument(description: 'Validates the integrity of existing migration files by checking if they have been tampered with.')]
+        bool $validate = false,
+    ): ExitCode {
+        if ($validate) {
+            $validationSuccess = $this->console->call('migrate:validate');
```

Comment:
```suggestion
            $validationSuccess = $this->console->call(MigrateValidateCommand::class);
```

Nitpick: prefer the FQCN here (previous code doesn't use that yet because support for FQCN was recently introduced)

Reply:
Oh, I didn't know this was added. I **much** prefer it like this also. I'll make sure to update it.

Reply:
Pushed the changes.